src/PIL/PngImagePlugin.py

```diff
@@ -868,6 +868,6 @@ def append(fp, cid, *data):
 Image.register_open(PngImageFile.format, PngImageFile, _accept)
 Image.register_save(PngImageFile.format, _save)
 
-Image.register_extension(PngImageFile.format, ".png")
+Image.register_extension(PngImageFile.format, [".png", ".apng"])
```

Comment:
You will see that CI is failing. That's because for multiple extensions, it needs to be 'register_extensions' -
```suggestion
Image.register_extensions(PngImageFile.format, [".png", ".apng"])
```